apicurito/deploy/cluster_role.yaml

```diff
@@ -7,8 +7,10 @@ rules:
   - console.openshift.io
   resources:
   - consoleyamlsamples
+  - consolelinks
   verbs:
   - get
   - create
+  - list
   - update
   - delete
```

Comment:
@myeung18 for this operator CSV and _role.yaml generated from go code, so you need to add this in go code.